packages/contracts/hardhat.config-fork.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+module.exports = {
+  solidity: "0.7.4",
+  networks: {
+    hardhat: {
+      chainId: 20200406,
+      blockGasLimit: 12000000,
+      forking: {
+        url: "https://zkopru-testnet.sifnoc.net",
+        blockNumber: 95 // Migration step 19 complete as 'testnet' network.
```

Comment:
<table>
<tr>
	<td>blockNumber
	<td>migration file
	<td>description
<tr>
	<td>63
	<td>18_setup
	<td>deployed Zkopru Contract
<tr>
	<td>64
	<td>18_setup
	<td>setChallengePeriod as 30
<tr>
	<td>65
	<td>18_setup
	<td>register account to BurnAuction contract
<tr>
	<td>93
	<td>19_migration_dest
	<td>Deployed Zkopru2 contract as migration dest
<tr>
	<td>94
	<td>19_migration_dest
	<td>setChallengePeriod as 30
<tr>
	<td>95
	<td>19_migration_dest
	<td>register account to BurnAuction contract
</table>